beacon-chain/rpc/statefetcher/fetcher.go

```diff
@@ -16,7 +16,12 @@ import (
 	"github.com/prysmaticlabs/prysm/shared/bytesutil"
 )
 
-// StateFetcher is responsible for retrieving the BeaconState.
+// IStateFetcher is responsible for retrieving the BeaconState.
+type IStateFetcher interface {
```

Comment:
Typically we avoid calling interface the same as a struct with just an `I` prefix, unless there is no better name. For example, we try to call interfaces by verbs, such as `StateFetcher`, `SlashingChecker`, etc. and the structs using a noun instead